src/makeRobot.js

```diff
@@ -37,8 +37,88 @@
  *
  * @return {Robot}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
-}
+  const robot = {
+    rn: name,
+    cv: version,
+    wv: wheels,
+    x: 0,
+    y: 0,
+
+    goForward: function(value) {
+      if (!value) {
+        this.y += 1;
+      } else if (value > 0) {
+        this.y += value;
+      } else {
+        this.y = this.y;
+      }
+
+      return this;
+    },
```

Comment:
```suggestion
    goForward(value = 1) {
      if (value > 0) {
        this.y += value;
      }
      
      return this;
    },
```

you could make your movement methods simplier by setting default parameters